src/Microsoft.Extensions.ObjectPool/DefaultObjectPool.cs

```diff
@@ -42,8 +42,7 @@ bool IsDefaultPolicy()
 
         public override T Get()
         {
-            T item = _firstItem;
-
+            var item = _firstItem;
```

Comment:
Many of the changes here are stylistic - I got rid of the implicit conversions, and swapped to `var`.